monitoring-custom-metrics/README.md

```diff
@@ -2,189 +2,51 @@
-
-To enable sidecar injection for all the pods running in namespace ***default*** run the command below.
-
-```bash
-$ kubectl label namespace default istio-injection=enabled
-namespace "default" labeled
- ```
-- [Istio reference documentation - Deploy your application](https://istio.io/docs/setup/kubernetes/quick-start#deploy-your-application).
-
-It is needed to add the sidecar.istio.io/inject annotation with value true to the pod template spec to enable injection. Do this in your Deployment yaml.
-
-
-```yaml
-spec:
-  template:
-    metadata:
-      annotations:
-        sidecar.istio.io/inject: "true"
-```
-
-- [Installing the Istio Sidecar](https://istio.io/docs/setup/kubernetes/sidecar-injection/#policy).
-
-
-#### 8081, the excluded port for metrics
-
-Istio 0.8 provides a way to include those inbound ports to be redirected to Envoy.
-
-Envoy will capture only traffic on ports listed in containerPorts of the pod, `containerPort: 8080`, or the list of ports in the annotation "traffic.sidecar.istio.io/includeInboundPorts".
-
-In Kyma port ***8080*** is used by the [Kubeless lambda functions](../event-subscription/lambda/README.md), therefore needs to be included in the inbound ports list.
-
-```yaml
-spec:
-  template:
-    metadata:
-      annotations:
-        traffic.sidecar.istio.io/includeInboundPorts: "8080"
-```
-
-**All other traffic bypasses Envoy** and goes straight to the container. In Kyma, use the suggested port 8081 to expose metrics.
-
 ### Expose a sample metrics application
 
-In order to expose prometheus metrics in golang the prometheus community provides a library, [client_golang ](https://github.com/prometheus/client_golang).
-
-This example has taken a basic example where Gauge and Counter metrics are exported using the package prometheus.
-
-1. Build Docker Image
+- Deploy the application, service and servicemonitor
     ```bash
-    # (linux, mac) users
-    $ eval $(minikube docker-env)
-
-    # windows user powershell
-    > & minikube docker-env | Invoke-Expression
-
-    $ kubectl config use-context minikube
-
-    $ cd examples/monitoring-custom-metrics
-
-    $ docker build -t sample-metrics-app -f Dockerfile .
-
-    ...
-    Successfully built b41158be6124
-    Successfully tagged sample-metrics-app:latest
+    kubectl apply -f deployment -R
     ```
-[Dockerfile here](Dockerfile)
+    
+#### Access the exposed metrics in Prometheus
 
-2. Deploy sample-metrics-app
+- Run the port-forward on service core-prometheus
+    
     ```bash
-    $ kubectl apply -f k8s/sample-metrics-app.yaml
-    service "metrics-8080" created
-    service "sample-metrics-8081" created
-    deployment.extensions "sample-metrics" created
-    servicemonitor.monitoring.coreos.com "metrics" created
+    kubectl port-forward -n kyma-system svc/core-prometheus 9090:9090
     ```
+In [targets](http://localhost:9090/targets#job-sample-metrics-8081) all the **sample-metrics** endpoint will appear.

```

Comment:
``All the **sample-metrics** endpoints appear as the [`Targets`](http://localhost:9090/targets#job-sample-metrics-8081) list.``

Reply:
done